### PR TITLE
Clarify domain guid

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/issues/510

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantSyncClientRestriction.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ Accept wildcard characters: False
 
 ### -DomainGuids
 
-Sets the domain GUID to add to the safe recipient list. Requires a minimum of 1 domain GUID. The maximum number of domain GUIDs allowed are 125.
+Sets the [domain GUID](/powershell/module/activedirectory/get-addomain) of the computer domain membership to add to the safe recipient list. Requires a minimum of 1 domain GUID. The maximum number of domain GUIDs allowed are 125.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantSyncClientRestriction.md:172]:
```suggestion
Sets the [domain GUID](/powershell/module/activedirectory/get-addomain) of the computer domain membership to add to the safe recipient list. Requires a minimum of one domain GUID. The maximum number of domain GUIDs allowed is 125.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by msbemba — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**